demo/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@mdx-js/monaco-demo",
+  "version": "0.0.0",
+  "type": "module",
+  "private": true,
+  "scripts": {
+    "start": "webpack serve"
+  },
+  "dependencies": {
+    "@mdx-js/monaco": "*",
+    "@types/webpack-env": "^1.0.0",
+    "css-loader": "^6.0.0",
+    "html-webpack-plugin": "^5.0.0",
+    "mini-css-extract-plugin": "^2.0.0",
+    "monaco-editor": "^0.34.0",
+    "webpack": "^5.0.0",
+    "webpack-cli": "^4.0.0",
+    "webpack-dev-server": "^4.0.0"
```

Comment:
it may be good to split `dependencies` vs `devDependencies`.
I tend to make anything that runs on the webpage `dependencies`, and make build tools and types `devDependencies`

Reply:
I did that too at first, then decided it’s pointless to make this distinction.

In my experience, because it doesn’t matter, at some point someone will put a dependency in the wrong place, or duplicate it into both `dependencies` and `devDependencies`.

In fact, if this would be published to npm as a built project, then they should all be in `devDependencies`.